pyspecdata/core.py

```diff
@@ -6714,6 +6716,16 @@ def _init_datanode(self,datanode,**kwargs):
         del self.h5file
         del self.datanode
         return
+    def __init__(self,pathstring,directory='.'):
+        self.pathstring = pathstring
+        #try:
+        self.h5file,self.datanode = h5nodebypath(pathstring,
+                check_only=True, directory=directory)
+        #except BaseException  as e:
+        #    raise IndexError("I can't find the node "+pathstring+explain_error(e))
+        print("about to call _init_datanode")
+        self._init_datanode(self.datanode)
+        atexit.register(self._cleanup)
```

Comment:
don't quite understand how this got moved around (though I probably did it?) .. I would move it up above init_datanode, so the diff is simpler

Reply:
Yes, this was your doing as I have not touched this (and diffed against core in master, it is exactly where you are saying to move it) -- so I have moved it there now

Reply:
yes, it looks like you don't have the most recent version of master.  PS, one of two ways of doing this -- 1) `git checkout master`, then `git pull`, then `git checkout alec_fitdata` or 2) `git fetch origin master:master` -- assuming you haven't made any changes to master locally, this will update it (note you can tab-complete the master:master, so it's not so much typing)

Reply:
Cool this is done